### DISCUSSION TITLE
Merge Blogger and Blogspot into Google?

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/tosdr.org#196] [Q&A]
Blogger and Blogspot should probably be merged as one service. But the privacy policy is the Google one, so should we merge that? Then what about Youtube for instance?

**Comment by michielbdejong** [comment]
For now, I'm adding the device fingerprinting point to all three, and then we can see how to deal with those duplicates if/when we merge.

**Comment by michielbdejong** [comment]
Actually, I can't find Blogger on phoenix, only Blogspot, so should see how to match that to this repo, where both services exist.

**Comment by hugoroy** [comment]
YT has additional  terms IIRC. 

Le 20 janvier 2018 14:53:31 GMT+00:00, Michiel de Jong <notifications@github.com> a écrit :
>Blogger and Blogspot should probably be merged as one service. But the
>privacy policy is the Google one, so should we merge that? Then what
>about Youtube for instance?

-- 
Envoyé de mon appareil mobile
Sent from my mobile device
